app/pipeline/manager/client.go

```diff
@@ -103,7 +113,7 @@ func (e *embedded) Detail(ctx context.Context, stage *drone.Stage) (*client.Cont
 		Netrc:   ConvertToDroneNetrc(details.Netrc),
 		System: &drone.System{
 			Proto: e.config.Server.HTTP.Proto,
-			Host:  "host.docker.internal",
+			Host:  getContainerHost(),
```

Comment:
as mentioned, derive both the `proto` and the `host` from `e.config.URL.Container`, and move the `getContainerHost` logic to `backfillURLs` in `cli/server/config.go`

Reply:
> as mentioned, derive both the `proto` and the `host` from `e.config.URL.Container`, and move the `getContainerHost` logic to `backfillURLs` in `cli/server/config.go`

fixed now